data/mods/Magiclysm/items/druid_items.json

```diff
@@ -0,0 +1,368 @@
+    "extend": { "qualities": [ [ "CUT", 2 ], [ "FABRIC_CUT", 2 ] ] },
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "watertight": true, "ammo_restriction": { "mag_awakened_water": 300 } } ],
+    "use_action": [
+      {
+        "target": "mag_awoken_weaver",
+        "msg": "The %s awakens.  It gazes at you with its many eyes, ready to serve you.",
+        "active": true,
+        "moves": 50,
+        "need_charges": 300,
+        "need_charges_msg": "The weaver does not have enough liquid.",
+        "type": "delayed_transform",
+        "transform_age": 432000,
+        "not_ready_msg": "The weaver isn't done growing yet."
+      },
+      {
+        "type": "repair_item",
+        "item_action_type": "repair_fabric",
+        "materials": [
+          "cotton",
+          "leather",
+          "lycra",
+          "nylon",
+          "wool",
+          "fur",
+          "faux_fur",
+          "nomex",
+          "kevlar",
+          "kevlar_layered",
+          "neoprene",
+          "parabolan_wool",
+          "gutskin",
+          "canvas",
+          "black_dragon_hide",
+          "demon_chitin"
+        ],
+        "skill": "tailor",
+        "tool_quality": 1,
+        "cost_scaling": 0.1,
+        "move_cost": 500
+      }
+    ]
+  },
+  {
+    "id": "mag_awoken_weaver",
+    "type": "ITEM",
+    "subtypes": [ "TOOL", "ARMOR", "ARTIFACT" ],
+    "name": { "str": "awoken wildwood weaver" },
+    "sub": "sewing_kit",
+    "description": "You never thought you'd see the day when a wildwood entity would be so playful and docile.  This weaver is ready to serve you, and can be used to repair clothing or create a shelter.",
+    "copy-from": "mag_adult_weaver",
+    "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "skills": [ { "value": "tailor", "add": 1 } ] } ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "watertight": true, "ammo_restriction": { "mag_awakened_water": 250 } } ],
+    "charges_per_use": 1,
+    "use_action": [
+      {
+        "type": "repair_item",
+        "item_action_type": "repair_fabric",
+        "materials": [
+          "cotton",
+          "leather",
+          "lycra",
+          "nylon",
+          "wool",
+          "fur",
+          "faux_fur",
+          "nomex",
+          "kevlar",
+          "kevlar_layered",
+          "neoprene",
+          "parabolan_wool",
+          "gutskin",
+          "canvas",
+          "black_dragon_hide",
+          "demon_chitin"
+        ],
+        "skill": "tailor",
+        "tool_quality": 1,
+        "cost_scaling": 0.1,
+        "move_cost": 300
+      },
+      [ "PLAY_GAME" ]
+    ]
+  },
+  {
+    "type": "ITEM",
+    "id": "mag_brokeweaver",
+    "subtypes": [ "ARTIFACT" ],
+    "symbol": ";",
+    "color": "green",
+    "name": { "str": "small damaged weaver" },
+    "description": "A tiny broken weaver.  It's legs bent and broken, it can no longer be used to repair clothing or create a shelter.",
```

Comment:
```suggestion
    "description": "A tiny broken weaver.  Its legs bent and broken, it can no longer be used to repair clothing or create a shelter.",
```